courses/free-courses-en.md

```diff
@@ -1610,6 +1610,8 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala & Functional Programming Essentials](https://www.udemy.com/course/rock-the-jvm-scala-for-beginners/)
+* [Scala Tutorial](https://youtu.be/-8V6bMjThNo?si=4QDqJthhYQiAjB02)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!